### PR TITLE
Add install method deployment

### PR DESCRIPTION
**Pull request by huikang**
split from #11 

Add install_method as a field to the rally deployment object

### CODE REVIEW

**Comment by huikang** [comment]
After this patch merged, we do not need install_method in the rally scenario file. The install_method is retried from the deployment. Please review the PR. Thanks.

File: rally_ovs/plugins/ovs/ovsclients_impl.py [hunk 1]
```diff
@@ -28,9 +28,6 @@ def create_client(self):
         return get_ssh_from_credential(self.credential)
 
 
-
```
Comment (l8huang) [inline rally_ovs/plugins/ovs/ovsclients_impl.py:31]: no valid changes(except whitespaces) made, this file should not be changed.

File: rally_ovs/plugins/ovs/ovsclients.py [hunk 2]
```diff
@@ -26,13 +26,11 @@ def configure(name):
     return plugin.configure(name, namespace=_NAMESPACE)
 
 
-
 class OvsClient(plugin.Plugin):
     def __init__(self, credential, cache_obj):
         self.credential = credential
         self.cache = cache_obj
 
-
```
Comment (l8huang) [inline rally_ovs/plugins/ovs/ovsclients.py:35]: no valid changes(except whitespaces) made, this file should not be changed.

File: rally_ovs/plugins/ovs/deployment/providers/ovn_sandbox_provider.py [hunk 3]
```diff
@@ -59,7 +59,6 @@ class OvsSandboxProvider(provider.ProviderFactory):
     }
 
 
-
```
Comment (l8huang) [inline rally_ovs/plugins/ovs/deployment/providers/ovn_sandbox_provider.py:62]: no valid changes(except whitespaces) made, this file should not be changed.

File: rally_ovs/plugins/ovs/consts.py [hunk 4]
```diff
@@ -22,5 +22,3 @@ class _ResourceType(utils.ImmutableMixin, utils.EnumMixin):
     SANDBOXES = "sandboxes"
 
 ResourceType = _ResourceType()
-
```
Comment (l8huang) [inline rally_ovs/plugins/ovs/consts.py:25]: no valid changes(except whitespaces) made, this file should not be changed.

File: rally_ovs/plugins/ovs/scenarios/ovn_network.py [hunk 5]
```diff
@@ -21,7 +21,6 @@
 LOG = logging.getLogger(__name__)
 
 
-
```
Comment (l8huang) [inline rally_ovs/plugins/ovs/scenarios/ovn_network.py:24]: no valid changes(except whitespaces) made, this file should not be changed.

**Comment by huikang** [comment]
removed the empty lines.

**Pull request merged**